### DISCUSSION TITLE
Stripe with Next js

### DISCUSSION THREAD

**Discussion by iampawanpoudel** [vercel/next.js#56097] [Help]
### Summary

How can I integrate stripe with next js for one time payment? I want to make user to able to update their free token using stripe and when user make a successful payment I want to increase their db token.

**Comment by cblberlin** [comment]
at your database create token field under User, then at stripe you can either configure a product table then embed it, or you can create the product directly in src code, then write some logic, when a payment successed, then update your token at your db
